src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -6,21 +6,26 @@ import session from 'services/session';
 import user from 'services/user';
 import { gordonColors } from 'theme';
 import styles from '../Doughnut.module.css';
+import { useTheme } from '@emotion/react';
 
 const lowBalance = 20; //dollars
 const reallyLowBalance = 10; //dollars
 
-let daysColor = gordonColors.primary.blue;
-let swipesColor = gordonColors.secondary.green;
-let dollarsColor = gordonColors.secondary.yellow;
-let guestColor = gordonColors.secondary.orange;
-let emptyColor = gordonColors.neutral.lightGray;
-
 const DiningBalance = () => {
   const [loading, setLoading] = useState(true);
   const [diningInfo, setDiningInfo] = useState(null);
   const [[daysRemaining, daysInSession], setDaysLeft] = useState([null, null]);
 
+  const getColor = (cssVar) => getComputedStyle(document.documentElement).getPropertyValue(cssVar);
```

Comment:
As above, this is not a good final solution, so I don't think we should merge it.

Reply:
Fair , I removed all uses of getComputedStyle for now